Makefile

```diff
@@ -1,5 +1,6 @@
+# Y - in response to Would you like to add PhraseApp configuration?
 create:
-	rails new $(APP_NAME) -m ./template.rb -T ${OPTIONS}
+	printf "Y\n" | rails new $(APP_NAME) -m ./template.rb -T ${OPTIONS}
```

Comment:
A reminder of `create_web` and `create_app` 😅

Reply:
Thanks for the reminder @rafayet-monon :D As we don't know which PR can merge first so once your PR is merged first I will rebase this one :D